newIDE/app/src/ObjectEditor/Editors/Model3DEditor.js

```diff
@@ -196,88 +280,493 @@ const Model3DEditor = ({
+                                    gdevelopTheme.list.itemsBackgroundColor,
+                                }}
+                              >
+                                <Line noMargin expand alignItems="center">
+                                  {connectDragSource(
+                                    <span>
+                                      <Spacer />
+                                      <DragHandleIcon />
+                                      <Spacer />
+                                    </span>
+                                  )}
+                                  <Text noMargin noShrink>
+                                    <Trans>Animation #{animationIndex}</Trans>
+                                  </Text>
+                                  <Spacer />
+                                  <SemiControlledTextField
+                                    margin="none"
+                                    commitOnBlur
+                                    errorText={nameErrors[animation.ptr]}
+                                    translatableHintText={t`Optional animation name`}
+                                    value={animation.getName()}
+                                    onChange={text =>
+                                      changeAnimationName(animationIndex, text)
+                                    }
+                                    fullWidth
+                                  />
+                                  <IconButton
+                                    size="small"
+                                    onClick={() =>
+                                      removeAnimation(animationIndex)
+                                    }
+                                  >
+                                    <Trash />
+                                  </IconButton>
+                                </Line>
+                                <Spacer />
+                              </div>
+                              <Spacer />
+                              <ColumnStackLayout expand>
+                                <SelectField
+                                  id="animation-source-field"
+                                  value={animation.getSource()}
+                                  onChange={(event, value) => {
+                                    animation.setSource(event.target.value);
+                                    forceUpdate();
+                                  }}
+                                  margin="dense"
+                                  fullWidth
+                                  floatingLabelText={
+                                    <Trans>GLB animation name</Trans>
+                                  }
+                                  translatableHintText={t`Choose an animation`}
+                                >
+                                  {sourceSelectOptions}
+                                </SelectField>
+                                <Checkbox
```

Comment:
Here you could use the same thing as in the component `DirectionTools`: an InlineCheckbox

Reply:
I fixed the case, is there any other difference?

Reply:
The text size I think. And it was for consistency's sake

Reply:
Indeed.
I'll keep it this way because if it's on its own line, it should be like other boolean properties.